plugins/repository-azure/src/main/java/org/elasticsearch/repositories/azure/AzureRepositoryPlugin.java

```diff
@@ -80,15 +74,6 @@ AzureStorageService createAzureStoreService(final Settings settings) {
         );
     }
 
-    @Override
-    public List<ExecutorBuilder<?>> getExecutorBuilders(Settings settings) {
```

Comment:
No need to deprecate here. We have never documented this threadpool and it was always seen as an internal setting.
Having it set will not prevent ES from starting up cleanly so I don't see anyone getting hurt here.